src/core/src/Environment.C

```diff
@@ -1491,26 +1484,27 @@ FullEnvironment::construct (RawType_MPI_Comm inputComm,
 FullEnvironment::FullEnvironment(
   const char*                    passedOptionsInputFileName,
   const char*                    prefix,
-  EnvOptionsValues* alternativeOptionsValues)
+  EnvOptionsValues* initialOptionsValues)
```

Comment:
I like the name changes.  "Self-documenting code" rather than "atavistic deceptive code" is a big plus.